packages/ses/src/error/tame-console.js

```diff
@@ -37,5 +38,34 @@ export const tameConsole = (
     };
   }
   const causalConsole = makeCausalConsole(originalConsole, loggedErrorHandler);
+
+  // Attach platform-specific error traps such that any error that bottoms out
+  // an event will get unwrapped by the causal console, revealing the error
+  // stack to the console for debugging.
```

Comment:
By "event" do you mean "turn"? I haven't been using the term "unwrapped", though I don't have an alternative on top of my head. The console reveals not just the stack but the full unredacted detailed error message, and attached notes, including their errors, etc...


```suggestion
  // Attach platform-specific error traps such that any error that gets thrown at top-of-turn (i.e., bottom of stack) will 
  // get logged by our causal console, revealing the diagnostic information associated with the error, including the
  // stack from when the error was created.
```

Reply:
I’ve integrated this change.